src/sagas.js

```diff
@@ -254,6 +265,15 @@ function* fetchConsoleVmMeta ({ vmId }) {
   return []
 }
 
+function* fetchVmSessions ({ vmId }) {
+  const sessions = yield callExternalAction('sessions', Api.sessions, { payload: { vmId } })
+
+  if (sessions && sessions['session']) {
+    return Api.sessionsToInternal({ sessions })
+  }
+  return []
+}
+
 function* getConsoleVm (action) {
```

Comment:
Please refactor (see comment above)

Use downloadVmConsole instead of getConsoleVm().